glyphs-reader/src/font.rs

```diff
@@ -482,17 +531,50 @@ pub struct RawGlyph {
 
 #[derive(Default, Clone, Debug, PartialEq, FromPlist)]
 pub struct RawLayer {
+    pub name: String,
     pub layer_id: String,
     pub associated_master_id: Option<String>,
     pub width: OrderedFloat<f64>,
     shapes: Vec<RawShape>,
     paths: Vec<Path>,
     components: Vec<Component>,
     pub anchors: Vec<RawAnchor>,
+    #[fromplist(alt_name = "attr")]
+    pub attributes: LayerAttributes,
     #[fromplist(ignore)]
     pub other_stuff: BTreeMap<String, Plist>,
 }
 
+impl RawLayer {
+    /// Return true if the layer is a draft that is not meant to be compiled.
+    ///
+    /// The presence of an associated master indicates this is not a simple 'master' instance.
+    /// Without 'attributes' that specify whether it's a special intermediate, alternate or
+    /// color layer, we can assume the non-master layer is a draft.
+    fn is_draft(&self) -> bool {
```

Comment:
I would guess this corresponds to py code that does the same, can we reference that py code?

Reply:
no, there isn't one. The inline comment above comes from observing that upon creating a new layer from Glyphs.app and immediately saving the file, you get exactly that: a layer with a not-None associated_master_id and no attributes set (and a name that is the current timestamp indicating a temporary draft). I don't think glyphsLib needs to explicitly filter "draft" layers as such, it just ignores them if they aren't master or one of the non-master attributes-bearing layers. Here I needed a way to exclude them when turning RawLayer instances into Layer ones so I created this `is_draft` to make the code more readable.